src/lmms_engine/parallel/expert_parallel/qwen3_moe/style.py

```
@@ -0,0 +1,127 @@
+        input_layouts: Optional[Placement] = None,
+        output_layouts: Optional[Placement] = None,
+        use_local_output: bool = True,
+    ) -> None:
+        super().__init__()
+        self.input_layouts = (input_layouts or Replicate(),)
+        self.output_layouts = (output_layouts or Replicate(),)
+        self.use_local_output = use_local_output
+        self.desired_input_layouts = (Replicate(),)
+
+    @staticmethod
+    def _partition_fn(name, mod, device_mesh):
```

Ruff (0.12.2)

Unused static method argument: `name`

(ARG004)